README.md

```diff
@@ -10,6 +10,13 @@ Our lecturer is [Nicolai Parlog](https://github.com/nicolaiparlog).
 
 We have to solve the [tasks](https://olat.vcrp.de/auth/RepositoryEntry/1676804160/CourseNode/95999652079122) in the limited period from 30.07.2017 (18:00) to 13.08.2017 (18:00).
 
+## Application
+To start our application first you have to open command prompt or Windows PowerShell. Navigate to the directory the jar-file is in. There you can run the following command:
+'''
```

Comment:
I assume you're going for a fenced code block here - they need to start with three backticks (not apostrophes):

    ```
    code goes here
    ```

It is also customary (maybe even necessary) to surround the block with empty lines.